kotlin/templates/libraries/jvm-okhttp/infrastructure/ApiClient.kt.mustache

```diff
@@ -305,6 +306,9 @@ import kotlinx.coroutines.delay
         if (requestConfig.headers[Accept].isNullOrEmpty()) {
             requestConfig.headers[Accept] = JsonMediaType
         }
+        if (requestConfig.headers["svix-req-id"].isNullOrEmpty()) {
+            requestConfig.headers["svix-req-id"] = Math.abs(Random.nextBits(32)).toString()
+        }
```

Comment:
Why not put it near line 354?

Reply:
Do you mean after the retries? The header wouldn't be added to the request in that case

Reply:
I meant added once, like with the other libs, rather on every loop.